tools/clang/test/DXC/dumpPSV_MS.hlsl

```diff
@@ -189,7 +189,7 @@ void main(
       op.normal = mpl.normal;
       op.malnor = gsMem[tig / 3 + 1];
       op.alnorm = mpl.alnorm;
-      op.ormaln = mpl.ormaln;
+      op.ormaln = mpl.ormaln + vid;
```

Comment:
I see many changes inside conditionals above.  Are they really all covered by this one test change?

Reply:
This issue only affects mesh shader primitive output.
The result for other tests will be the same.